### ISSUE TITLE
some solution so that narrow windows doesn't result in categories being hidden

### ISSUE THREAD

**Issue by jonorthwash** [sushain97/apertium-on-github#22]
Would be nice if we could have one of these solutions or something similar:

**Option 1**: Have the headings fix to top and bottom of screen when not in viewport, so it's clear that there are other categories down (or up).

**Option 2**: Have something floating (say on the top right) indicating that there are other categories up or down, and lets you click to jump to them.  If all the category headings are in the viewport, don't show this.

**Comment by sushain97** [comment]
https://atomiks.github.io/30-seconds-of-css/#overflow-scroll-gradient might be cool

**Comment by jonorthwash** [comment]
That would show that there are more items in the categories that are visible, but it wouldn't help you to know that there's another category at the bottom.

In case it's not clear why this is a problem, here's how it renders in a realistically sized browser window:
![screenshot-2018-2-27 apertium source browser](https://user-images.githubusercontent.com/963849/36751463-428943d6-1bce-11e8-9a9d-1d9107406207.png)
It's not clear that there are more sections of repos listed below this.

**Comment by sushain97** [comment]
Indeed, it would not solve all the problems. Realistically sized browser is
quite subjective :)

**Comment by jonorthwash** [comment]
Note that this issue is about not knowing there are more categories below—not about needing to know that you can still scroll down.  The solution you proposed doesn't address that.

**Comment by sushain97** [comment]
Indeed. It's more something I stumbled on that seemed potentially useful and related.

**Comment by sushain97** [comment]
@jonorthwash what do you think about having a "menu" sort of line immediately below the Apertium/GitHub logo bar that has anchor links to each of the sections? potentially, the visible links could be bolded or something

**Comment by jonorthwash** [comment]
Would be better than nothing, though not long-term ideal.

**Comment by jonorthwash** [comment]
Update of current view:
![2018-09-06-23 09 58_001](https://user-images.githubusercontent.com/963849/45196406-06f6ac80-b22a-11e8-87a2-fd6069aef7f6.png)

**Comment by jonorthwash** [comment]
I still think the best solution is to have anything offscreen to hover at the top or bottom of the screen (depending on which direction off screen it is), and when your scrolling catches up with the text, have it display normally.

Examples of similar things include the [MediaWiki debugging toolbar](https://www.mediawiki.org/wiki/Debugging_toolbar) and [this site](https://languageworkshop.indiana.edu/) (especially at full size).

**Comment by dolphingarlic** [comment]
Is the desire to have something like in [this](https://www.w3schools.com/howto/howto_js_navbar_sticky.asp), or is it something more complicated?

**Comment by jonorthwash** [comment]
It's a little more complicated, because there can potentially be things off-screen in both directions.
